apps/api-portal/pages/index.tsx

```diff
@@ -6,6 +7,8 @@ import AccessTokenCreate from 'components/access-token/access-token-create/acces
 import RequireAdditionalInfo from 'components/require-additional-info/require-additional-info'
 import styles from '../styles/index.module.css'
 
+path.resolve('./next.config.js')
```

Comment:
is this coming from https://github.com/nrwl/nx/issues/9017#issuecomment-1140066503 ? I'd add a comment to understand why this is needed